doc/source/index.rst

```diff
@@ -142,7 +142,9 @@ Academic Papers
 
    cluster/index.rst
    cluster/launcher.rst
+   cluster/cloud.rst
    cluster/deploy.rst
+   cluster/config.rst
```

Comment:
RE: config to be top level, I'm not sure this is worth it. It's only applicable to the cluster launcher (not for cluster managers or on-prem clusters), and can be easily accessed in the launcher page. The vertical tradeoff doesn't seem worth it.

Reply:
ok, I will change this. Thanks.